### ISSUE TITLE
TypeError Python 2.7 

### ISSUE THREAD

**Issue by vladtf** [vgrem/Office365-REST-Python-Client#473]
Hi, I was trying to use Office365-REST-Python-Client v2.3.10 (latest released version) with Python 2.7.18, but I get a TypeError:

```
Traceback (most recent call last):
  File "downloadFileExample.py", line 28, in <module>
    file = ctx.web.get_file_by_server_relative_path(file_path).download(local_file).execute_query()
  File "/home/vladtf/.local/lib/python2.7/site-packages/office365/sharepoint/client_context.py", line 264, in web
    self.__web = Web(self)
  File "/home/vladtf/.local/lib/python2.7/site-packages/office365/sharepoint/webs/web.py", line 63, in __init__
    resource_path = ResourcePath("Web")
  File "/home/vladtf/.local/lib/python2.7/site-packages/office365/runtime/paths/resource_path.py", line 13, in __init__
    super().__init__(parent)
TypeError: super() takes at least 1 argument (0 given)
```

With version of Office365-REST-Python-Client v2.3.9 and Python 2.7.18 I have no errors.

The issue seem to be from: [resource_path.py](https://github.com/vgrem/Office365-REST-Python-Client/blob/master/office365/runtime/paths/resource_path.py#L13)

**Comment by vgrem** [comment]
Greetings, 

thank you for catching it, the fix will be included in the next release.

**Comment by vladtf** [comment]
Thanks for fast response!

**Comment by vgrem** [comment]
Has been resolved in [`2.3.11` release](https://github.com/vgrem/Office365-REST-Python-Client/releases/tag/2.3.11)
